tests/torchtune/modules/optim/test_cpu_offload.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import copy
+
+import pytest
+import torch
+from torch import nn
+from torchtune.modules.optim import CPUOffloadOptimizer
+from torchtune.modules.optim._cpu_offload import _CPU_OFFLOAD_OPTIM_AVAILABLE
+
+if not _CPU_OFFLOAD_OPTIM_AVAILABLE:
+    pytest.skip("CPU offload optimizer is not available", allow_module_level=True)
+
+
+class TestCPUOffloadOptimizer:
+    @pytest.mark.skipif(
+        not torch.cuda.is_available(), reason="CPU offload optimizer requires CUDA"
+    )
+    @pytest.mark.parametrize("offload_gradients", [False, True])
+    def test_cpu_offload_optimizer(self, offload_gradients):
+        model_ref = nn.Sequential(
+            nn.Linear(128, 256), nn.GELU(), nn.Linear(256, 128)
```

Comment:
for the sake of speed, would it make sense to make this smaller, e.g. 8, 16? I guess this shouldnt impact the test robustness. Is that right?